model/registry/db.yaml

```diff
@@ -229,7 +237,8 @@ groups:
               stability: experimental
             - id: mssqlcompact
               value: 'mssqlcompact'
-              brief: 'Microsoft SQL Server Compact'
+              deprecated: "Removed."
```

Comment:
Maybe mention that `other_sql` can be used? Same for some of the other values here.